examples/Cargo.toml

```diff
@@ -8,11 +8,11 @@ edition = "2018"
 # [dependencies] instead.
 [dev-dependencies]
 tower = { version = "0.4", path = "../tower", features = ["full"] }
-tower-service = "0.3" 
+tower-service = "0.3"
 tokio = { version = "1.0", features = ["full"] }
 rand = "0.8"
 pin-project = "1.0"
-futures = "0.3"
+futures = "^0.3.22"
```

Comment:
You can drop the leading caret. That's already the implicit behavior of Cargo's version resolver.